<launch>
  <!-- Which estimator to use -->
  <arg name="pose_estimator" value="gazebo_pose_estimator_node" />

  <!-- Load parameters onto server for object config -->
  <rosparam file="$(find pose_estimator)/config/objects.yaml" command="load" />
  <param name="/pose_estimator_topic" value="/pose_estimator" />

  <!-- Launch selected node -->
  <node name="gazebo_pose_estimator_node" pkg="pose_estimator" type="$(arg pose_estimator)" />
</launch>
